templates/options.erb

```diff
@@ -0,0 +1,15 @@
+Dpkg::Options {
+<%- @_options.each_pair do |config, value| 
+  if %w(force_confdef force_confold force_confnew force_confmiss).include?(config) then
+    if value.is_a?(TrueClass) || value.is_a?(FalseClass) then 
+      if value then -%>
```

Comment:
We check if it is a boolean or not so that we can provide a usefull error message to the user i.e. this value must be a boolean.  We then we check if the value is true.